rest-api-spec/src/main/resources/rest-api-spec/api/downsample.downsample.json

```diff
@@ -1,8 +1,8 @@
 {
-  "rollup.rollup":{
+  "downsample.downsample":{
```

Comment:
What does this do to our client friends?

Reply:
Clients  should probably be modified so that they invokes the `downsample` endpoint.

I have already tagged the @elastic/es-clients team in this PR. I will reach out to them and give them a heads up about this change.

Reply:
If the only API rename is `rollup.rollup` into `downsample.downsample` this should not be an issue since this API is `experimental` and we can have BC break.

Reply:
@ezimuel thanks for confirming. The detailed list of changes introduced can be found in the description of this PR. Regarding the API this is the only change.